prow/scripts/cluster-integration/helpers/cleanup-gardener.sh

```diff
@@ -31,7 +31,6 @@ requiredVars=(
 
 utils::check_required_vars "${requiredVars[@]}"
 
-export GARDENER_PROJECT_NAME=${GARDENER_KYMA_PROW_PROJECT_NAME}
 export GARDENER_CREDENTIALS=${GARDENER_KYMA_PROW_KUBECONFIG}
```

Comment:
You are using `GARDENER_KYMA_PROW_KUBECONFIG` env here. Is this export still needed?